modules/dxp/apps/search-experiences/search-experiences-service/src/main/resources/com/liferay/search/experiences/internal/model/listener/dependencies/rescore_by_text_embedding.json

```diff
@@ -70,7 +70,7 @@
 						},
 						{
 							"defaultValue": "cosineSimilarity",
-							"helpText": "Vector functions measure similarity between search keywords and target document text embeddings. Check model documentation for suitability.",
+							"helpText": "Specifies the vector function for measuring similarity between the text embeddings of search keywords and target document. Check model documentation for suitability.",
```

Comment:
@peerkar Do you like the use of an action verb directed at the user (like Specify, Set, Choose, etc.) to start the help text? Because the original (I think it's BChan's edit?) was shorter and was a complete sentence (the verb in the old text was "measure").  

I usually do what you've done in System Settings descriptions.

I'm not sure if we need to do it to the element tooltips (help text) though. They can just be descriptive complete sentences.

I just wasn't sure if you like them better this way or if it's just to satisfy BChan.

Reply:
BChan's wordsmithed:
> Vector functions measure similarity between the text embeddings of search keywords and target documents. Check the model's documentation for suitability.

Petteri's wordsmithed:
> Specify the vector function for measuring similarity between the text embeddings of the search keywords and target documents. Check the model's documentation for suitability.",

Reply:
I'm not even completely sure about the conventions in portal. I tried to look around in Language.properties and there seems to be several different flavors of help texts, like:

- **Actionable advices, what I as a user should do**: "Enter the name of the indexed field to aggregate results by."
- **What the configuration does:** "This configuration only applies to EMBEDDED (Development) mode. Set custom settings for the sidecar Elasticsearch node, in YML format (elasticsearch.yml)."
  **What just might happen if xyz**: "Selecting this will aggregate all sites to one report in Analytics Cloud."

Personally I probably generally find an actionable advice the best experience. When it starts with "Select xyz ... " it's personal and kind of empowering. When its starts with "Configuring xyz will ... " the sentiment is a bit like  "Pushing this button might break everything. Figure it out. Good luck"

I'll try with "Petteri's wordsmithed". 

Btw I checked help texts in some of our other elements and maybe some wordsmithing there could also be good. There are texts without verbs, too.

Reply:
Thanks @rbohl !